lib/maze/server.rb

```diff
@@ -124,7 +124,7 @@ def start
             server.mount '/notify', Servlet, errors
             server.mount '/sessions', Servlet, sessions
             server.mount '/builds', Servlet, builds
```

Comment:
Should Bugsnag-Integrity checks be enabled for requests made to the /build endpoint?

Reply:
There are now a few levels of control now as to when the integrity header will be enforced.  This is the most blunt level of control before it flows down to the MR servlet (which will not enforce it for `multipart/form-data` requests).  Whilst I wouldn't currently expect any build requests to have the integrity header, I think it's reasonable to leave it to the downstream mechanisms to control because, in theory, the product change change and include the digest.  Log requests, however, are not a part of the product and can therefore be blanket excluded.